source/templates/actions.md

```diff
@@ -117,7 +117,7 @@ When an action is triggered, but no matching action handler is
 implemented on the controller, the current route, or any of the
 current route's ancestors, an error will be thrown.
 
-![Action Bubbling](/images/template-guide/action-bubbling.png)
+<img src="/images/template-guide/action-bubbling.png" alt="Action Bubbling">
```

Comment:
You mean, back the markdown? (Both versions use relative links, both img tag and markdown version).

Not sure why that link is not working on the actual live site because it is not broken when I build it with:

-![Action Bubbling](/images/template-guide/action-bubbling.png)